olexandr.vakulinskiy/PR1/practice.rb

```diff
@@ -0,0 +1,21 @@
+require 'faraday'
+
+class Image
+  def download(url, file_name)
+    response = Faraday.get url
+    url_ext = response.headers['content-type'].split('/')[-1]
+
+    raise ArgumentError, "wrong url, status code #{response.status}" unless response.status == 200
+    raise TypeError, 'file is not an image' unless image?(url_ext)
+
+    File.open("#{file_name}.#{url_ext}", 'wb') { |fp| fp.write(response.body) }
+  end
+
+  def image?(url_ext)
+    %w[jpg jpeg png].include?(url_ext)
```

Comment:
Actually there are much more image extensions, it's better to check the first part of content_type to be equal to image.

Reply:
Ok, I'll fix it, it will be fast